superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -35,10 +35,15 @@ import {
   SupersetTheme,
   t,
   VizType,
+  useTheme,
 } from '@superset-ui/core';
 import rison from 'rison';
 import { useSingleViewResource } from 'src/views/CRUD/hooks';
 
+import { AntdForm } from 'src/components';
```

Comment:
Can we use `src/components/Form` instead? We are gradually removing these exports.

Reply:
For some reason Form.List isn't working, going to look into it some more but not sure why it's bonking out